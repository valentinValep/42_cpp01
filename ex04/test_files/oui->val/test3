ouinon oui ou bien non, perso plutot oui
..euh en fait non. Ah quoi que oui
Ok j'ai decide, 'non'
Je reviens sur ce que j'ai dit, "oui".
