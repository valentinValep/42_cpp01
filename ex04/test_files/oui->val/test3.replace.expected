valnon val ou bien non, perso plutot val
..euh en fait non. Ah quoi que val
Ok j'ai decide, 'non'
Je reviens sur ce que j'ai dit, "val".
